types/axios-concurrency/axios-concurrency-tests.ts

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { ConcurrencyManager, ConcurrencyManagerInstance } from "axios-concurrency";
+
+const axiosInstance = axios.create({});
+
```

Comment:
There should really be at least one *passing* test here

Reply:
Could the below code be a passing test?

```
const manager: ConcurrencyManagerInstance = ConcurrencyManager(axiosInstance, 1);

// $ExpectType void
manager.detach();
```